bigchaindb/backend/mongodb/query.py

```diff
@@ -153,13 +153,17 @@ def get_spent(conn, transaction_id, output):
     cursor = conn.run(
         conn.collection('bigchain').aggregate([
             {'$match': {
-                'block.transactions.inputs.fulfills.txid': transaction_id,
-                'block.transactions.inputs.fulfills.output': output
+                'block.transactions.inputs.fulfills': {
```

Comment:
@jmduque suggested giving every output a single unique ID, rather than using `transation_id` and `output`. For example, one could use a concatenation of `transaction_id + '_' + str(output)`. Would doing that solve the problem?